WooCommerce/WooCommerceTests/Notifications/PushNotificationsManagerTests.swift

```diff
@@ -42,7 +42,11 @@ final class PushNotificationsManagerTests: XCTestCase {
         defaults = UserDefaults(suiteName: Sample.defaultSuiteName)
         defaults.removePersistentDomain(forName: Sample.defaultSuiteName)
 
+        // Most of the test cases expect a nil site ID, otherwise the dispatched actions would not match.
         storesManager = MockupStoresManager(sessionManager: .testingInstance)
+        storesManager.sessionManager.setStoreId(nil)
+        ServiceLocator.setStores(storesManager)
```

Comment:
When I removed this, the tests still passed. 🤔 Maybe it's not needed?

```suggestion

```

Reply:
I struggled with these app singletons (`storesManager` and `sessionManager`) for quite a while before making the PR 😓 

thanks to your suggestions below about using `storesManager` from `configuration` instead of the global singleton, it's indeed not needed anymore! removed in https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/7620bb0da4f901ebfd141939e5511a9ab657c39e